packages/react-components/react-color-picker-preview/stories/src/ColorPicker/ColorPickerDefault.stories.tsx

```diff
@@ -91,6 +94,26 @@ export const Default = () => {
     }
   };
 
+  const onAlphaChange: SpinButtonProps['onChange'] = React.useCallback(
+    (_ev, data) => {
+      const value = data.value ?? (data.displayValue !== undefined ? parseFloat(data.displayValue) : null);
+
+      if (value === null || Number.isNaN(value) || !/^(0|1|0?\.\d+|1\.0?)$/.test(value.toString())) {
+        return;
+      }
```

Comment:
We could probably simplify this a bit:

```suggestion
      const value = data.value ?? parseFloat(data.displayValue ?? 
      '');

      if (Number.isNaN(value) || value < 0 || value > 1) {
        return;
      }
```